src/jit.rs

```diff
@@ -206,8 +206,9 @@ const ANCHOR_CALL_UNSUPPORTED_INSTRUCTION: usize = 11;
 const ANCHOR_EXTERNAL_FUNCTION_CALL: usize = 12;
 const ANCHOR_INTERNAL_FUNCTION_CALL_PROLOGUE: usize = 13;
 const ANCHOR_INTERNAL_FUNCTION_CALL_REG: usize = 14;
-const ANCHOR_TRANSLATE_MEMORY_ADDRESS: usize = 21;
-const ANCHOR_COUNT: usize = 34; // Update me when adding or removing anchors
+const ANCHOR_VALIDATE_AND_PROFILE_INSTRUCTION_COUNT: usize = 21;
+const ANCHOR_TRANSLATE_MEMORY_ADDRESS: usize = 22;
+const ANCHOR_COUNT: usize = 35; // Update me when adding or removing anchors
```

Comment:
I realized I didn't increment the Anchor here: https://github.com/solana-labs/rbpf/pull/629/files#diff-7d39efd566d08b201d1e747f11b3909be0d39fd3d4c89cfd28e0b21bab3ec27a

Should this be 36?